makefiles/Makefile.docs-bi-connector

```diff
@@ -38,6 +39,8 @@ next-gen-html:
 	npm run build; \
 	cp -r "${REPO_DIR}/snooty/public" ${REPO_DIR};
 
+STABLE_BRANCH_NEXT_GEN = STABLE_BRANCH=`grep 'manual' ${REPO_DIR}/published-branches.yaml | cut -d ':' -f 2 | grep -Eo '[0-9a-z.]+'`
```

Comment:
manual isn't a branch in this repo -- did you mean 'master'?

Reply:
the branch is indeed master, but it is pulled from looking at the manual key here https://github.com/mongodb/docs-worker-pool/blob/meta/publishedbranches/docs-bi-connector.yaml#L26